src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ReplyToMessage.vue

```diff
@@ -22,7 +23,12 @@
               ></v-textarea>
             </v-col>
           </v-row>
-          <FileUploader @update:files="handleFileUpdate" />
+          <FileUploader
+            ref="FileUploader"
+            :rules="[Rules.conditionalWrapper(attachments.length > maxNumberOfFiles, '')]"
```

Comment:
oh I don't think this rule is needed. 

You have it covered in the fileUploader with tooManyFiles

Reply:
removed.